pkg/client/restclient/OWNERS

```diff
@@ -0,0 +1,24 @@
+reviewers:
```

Comment:
I'm also the reviewer of the top-level folder, so I don't need to be reviewer of this subdirectory.

(@deads2k I talked with @apelisse offline. He said being a reviewer means one would review the details of PR, while being an approver means one would review the intent of the PR, so being an approver is not a superset of being an reviewer)